Jay-RecipesX/app/src/main/java/com/example/jay_recipesx/Core/AppFunctions/AppFunctions.kt

```diff
@@ -0,0 +1,20 @@
+package com.example.jay_recipesx.Core.AppFunctions
+
+class AppFunctions {
+    companion object {
```

Comment:
why class?

Reply:
This Class provide reusable functions to all over the app.

Reply:
why can't we just use function, and use it directly?